Sources/Plasma/PubUtilLib/plNetGameLib/Private/plNglAuth.cpp

```diff
@@ -2387,6 +2388,20 @@ static bool Recv_ScoreGetHighScoresReply(
     return true;
 }
 
+//============================================================================
+static bool Recv_ServerCaps(
+    const uint8_t   msg[],
+    unsigned        bytes,
+    void *          param
+    ) {
+    const Auth2Cli_ServerCaps & caps = *(const Auth2Cli_ServerCaps *)msg;
+
+    hsReadOnlyStream stream(caps.byteCount, reinterpret_cast<const void*>(caps.buffer));
+    ((CliAuConn*)param)->caps.Read(&stream);
```

Comment:
Agreed. There are plenty of places in the protocol where a duped size parameter is not unusual. Although a lot of those are actually unneeded :D